.prettierrc

```diff
@@ -14,9 +16,8 @@
         "tabWidth": 4,
         "useTabs": false,
         "singleQuote": false,
-        "bracketSpacing": false,
-        "explicitTypes": "always"
```

Comment:
I guess this applies for everything but *.sol ?

Reply:
It only applies to *.sol,
_explicitTypes_ is unsupported option for solidity linter.

Reply:
> I guess this applies for everything but *.sol ?

It only applies to *.sol,
_explicitTypes_ is unsupported option for solidity linter. 
@pavlovcik